src/SubscriptionQueries.ts

```diff
@@ -0,0 +1,55 @@
+const stateRootAvailableSubscriptionQuery = 'subscription{stateRootAvailables(orderDirection:'
+  + ' desc, first: 1){'
++ '   id'
++ '   contractAddress '
++ '  }'
++ '}';
+
+const SubscriptionQueries: Record<string, Record<string, string>> = {
+  origin: {
+    stakeRequesteds: 'subscription{stakeRequesteds(orderDirection: desc, first: 1){'
+    + '   id'
+    + '   contractAddress '
+    + '  }'
+    + '}',
+
+    stakeIntentDeclareds: 'subscription{stakeIntentDeclareds(orderDirection: desc, first: 1){'
+    + '   id'
+    + '   contractAddress '
+    + '  }'
+    + '}',
+
+    stateRootAvailables: stateRootAvailableSubscriptionQuery,
```

Comment:
Shouldn't be added here yet I guess. It might even fail as we haven't defined this entity in schema for origin sub graph yet.

Reply:
Done!